rest/api_test.go

```diff
@@ -7066,6 +7066,9 @@ func TestEnsureRevocationAfterDocMutation(t *testing.T) {
 	// Skip to seq 19 and then update doc foo
 	revocationTester.fillToSeq(19)
 	revID = rt.createDocReturnRev(t, "doc", revID, map[string]interface{}{"channels": "A"})
+	changes, err := rt.WaitForChanges(1, "/db/_changes?since=19&revocations=true", "user", true)
+	require.NoError(t, err)
+	assert.Equal(t, 1, len(changes.Results))
```

Comment:
Completely fine to leave in and doesn't hurt for extra safety but just FYI this assert should never fail.

WaitForChanges will error if the results != 1 and the require will mean it quits before it gets here 😄